pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -0,0 +1,79 @@
+package bootstrappolicy
+
+import (
+	"strings"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/controller"
+
+	"github.com/golang/glog"
+)
+
+const saRolePrefix = "system:controller:"
+
+var (
+	// controllerRoles is a slice of roles used for controllers
+	controllerRoles = []authorizationapi.ClusterRole{}
+	// controllerRoleBindings is a slice of roles used for controllers
+	controllerRoleBindings = []authorizationapi.ClusterRoleBinding{}
+)
+
+func addControllerRole(role authorizationapi.ClusterRole) {
```

Comment:
No new openshift roles in our code here.  Let's use the rbac roles and our converter.  It will make future unification easier.